curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64e4e7bbedb22d6939001ad3.md

```diff
@@ -7,9 +7,13 @@ dashedName: step-7
 
 # --description--
 
-Add a `click` event listener to the `closeTaskFormBtn` variable, then use the `showModal()` method on your `confirmCloseDialog` variable. This will display a modal with the `Discard` and `Cancel` buttons.
+The HTML `dialog` element has a `showModal()` method that can be used to display a modal dialog box on a web page.
 
-`showModal()` is a method associated with the HTML `dialog` element. It is used to display a modal dialog box on a web page.
+`dialogElement.showModal();`
```

Comment:
This should be inside a code fence like this 

````
```js
dialogElement.showModal();
```
````

Reply:
Done. Thanks and I apologize for the inconvenience caused.